ludwig/schema/utils.py

```diff
@@ -1,9 +1,11 @@
+import copy
```

Comment:
I've factored the changes from this PR out into the defaults mixin PR since they are required for that change: https://github.com/ludwig-ai/ludwig/pull/2628

Reply:
Need to keep the convert submodules and `__repr__` changes here need to stay in this PR since they are used in the config object.